### ISSUE TITLE
Include media controls

### ISSUE THREAD

**Issue by fdiogo** [fdiogo/bityn#4]
There is no way to pause, stop, go forward or backward and no volume controls. Include the media controls on the app.

**Comment by slayerjain** [comment]
I wanna try! :)

**Comment by slayerjain** [comment]
<img width="801" alt="screen shot 2016-10-21 at 12 33 45 am" src="https://cloud.githubusercontent.com/assets/12831254/19574030/2ec2143c-9726-11e6-81fb-3eb6eaa442d5.png">
This is what i see after running the app. is this normal?

**Comment by fdiogo** [comment]
The left part is the sidebar and the red the content section. Those two colors are just indicative. The app right now loads all the mp3 in the default Music folder and adds to the black sidebar. If it's empty you probably don't have anything there. Anyway, you can always press Ctrl + Shift + I to open DevTools and check for any error

**Comment by slayerjain** [comment]
my itunes library is full of music :/

**Comment by slayerjain** [comment]
also, Ctrl + Shift + I doesn't bring dev tools. I'm on OS X

**Comment by fdiogo** [comment]
Try Alt+Cmd+I. Can you somehow check what app.getPath('music') gives you? Try to console.log it or just debug (if you can bring up the devtools)

**Comment by fdiogo** [comment]
If this has got you stuck I can quickly patch this to prompt a dialog to choose the folder

**Comment by slayerjain** [comment]
I've not used electron before, but have used NWjs, but it's interesting and I'll learn it. Any help would still be appreciated.

**Comment by slayerjain** [comment]
as of now, how can i load some media files to it?

**Comment by fdiogo** [comment]
On [app.js](https://github.com/fdiogo/bityn/blob/master/app.js) line 16,17 there's the expression app.getPath('music'). This is platform agnostic and should return a valid path to the default music folder of you OS. I'm running Manjaro Linux and works well but I haven't tried on macos. If you could somehow tell me what that returns for you we could exclude some possibilities as the issue. But for now, the only way it loads is by having audio files in the default music folder of the OS.

**Comment by fdiogo** [comment]
Ok just pushed a change. Can you pull and try it out? :)

**Comment by karldreher** [comment]
@fdiogo Hi, I was able to replicate the issue and I think I can see at least part of the blank screen issue, it appears to be that audio-player was not declared in the manifest.  I will submit an initial pull request and work on this a little more after I learn more about this project.

**Comment by fdiogo** [comment]
You are absolutely right. Thanks for noticing it. Will wait for your PR :)

**Comment by karldreher** [comment]
@slayerjain  Respecting your desire to dig in on the code if you wish, but opening another PR for some basic transport controls.  This dosen't include volume control or stop, and has an integrated play/pause button.  

@fdiogo  I had these assets sitting around from a similar project from years ago, if they at least fill a spot to build a skeleton I thought they might help.  I also started some of the HTML and CSS.  PR incoming.

**Comment by slayerjain** [comment]
even after the addition of the folder chooser dialog i get a blank screen. :(

**Comment by fdiogo** [comment]
@slayerjain Did you run npm install before running again?

@karldreher I'll have a check right now :)

**Comment by slayerjain** [comment]
yes i ran npm install after merging

**Comment by fdiogo** [comment]
Try the following @slayerjain :
- Make sure you pull again, there are new changes
- On app.js:16 replace [app.getPath('music')] with just [ ].
- On launch select a folder you know has tracks
- If that does not work, open DevTools and tell me what you see there, or on the terminal used to launch the app.

**Comment by slayerjain** [comment]
The media files appeared now, but the files within sub folders aren't visible, and the background looks different now. :D

**Comment by slayerjain** [comment]
clicking on the media doesn't play it, and there's an error on the console that says, " Electron Helper[7104:354023] Couldn't set selectedTextBackgroundColor from default ()"

**Comment by fdiogo** [comment]
Yeah good catch, the search is not recursive! This whole issue brought up what apparently is an issue. We can't just add the default music folder, by default or the app bugs out if something is wrong with the directory. Thanks, will spawn two issues with this info :)

**Comment by slayerjain** [comment]
so no mp3 playback as of now?

**Comment by fdiogo** [comment]
Geez, how many errors has this app with macOS O.o Yes there is mp3 playback, but it seems you still have problems xD

**Comment by slayerjain** [comment]
what about a separate branch for macOS? :D

**Comment by fdiogo** [comment]
That shouldn't be necessary, this is Electron after all. Just have to be careful with the API and use it properly.

**Comment by slayerjain** [comment]
Oh yeah, that's the whole point!

**Comment by fdiogo** [comment]
Unfortunately I ran out of ideas for the moment. It's hard to pinpoint the error without having the problem in front of me. Could you try to debug or give me more info about this bug @slayerjain?

**Comment by slayerjain** [comment]
@fdiogo i'll look into it

**Comment by fdiogo** [comment]
Check the commit fdiogo@312d6dc07c352ecfe2d69700491293968bda92fa
